diffsims/tests/test_generators/test_sphere_mesh_generators.py

```diff
@@ -36,6 +37,16 @@ def test_random_sphere_mesh():
     assert grid.shape[0] == 10313
     assert grid.shape[1] == 3
 
+    res = 10
```

Comment:
this is the bit that will have a conflict with #143 - should be simple to resolve